lib/chef/provider/chef_organization.rb

```diff
@@ -52,7 +52,9 @@ def whyrun_supported?
     new_resource.members.each do |user|
       if !existing_members.include?(user)
         converge_by "Add #{user} to organization #{new_resource.name}" do
-          rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users/#{user}", {})
```

Comment:
@jkeiser I was getting a little lost when you said:
"The JSONOutput middleware has a global way to do that, through :raw_output: https://github.com/chef/chef/blob/master/lib/chef/http/json_output.rb#L55-L59"

but I am guessing you mean:
https://github.com/chef/chef/blob/master/lib/chef/http/json_output.rb#L52-L54